### ISSUE TITLE
Finding the Mean

### ISSUE THREAD

**Issue by ellihammons21** [DS4PS/cpp-529-spr-2021#5]
Can anyone tell me how to find the average house-price-to-income ratio across all US counties over the time period? Is this information included in the data frame? I tried running the base r mean function: 

mean(CenDF$HHInc_HousePrice_Ratio)

but that didn't work. I'm assuming it should be pretty easy to get this information from the dataframe, but I must be missing something.

Thanks!

**Comment by RTrehern-ASU** [comment]
@ellihammons21 ,

I don't know if the answer I came up with is correct or not, but I used the following function and got a numeric response:

```{r}
library(dplyr)
summarise(CenDF, Average = mean(HHInc_HousePrice_Ratio, na.rm = T))
```

**Comment by Johaning** [comment]
I also used na.rm = T, but the mean command like ellihammons21 used: 

mean( CenDF2012$HHInc_HousePrice_Ratio, na.rm = TRUE )

**Comment by Anthony-Howell-PhD** [comment]
Yes, when the variable has missing values, you need to add the na.rm=TRUE to let R know that you are aware of the missing values and you still want to get the mean.   Closing the case here.
